README.md

```diff
@@ -41,23 +47,58 @@ resolved Promise.
 
 ### Generate a DID Document
 
-Generate a new DID document.
+#### Generating and Registering a Veres One DID Document
 
-* options - a set of options used when generating the DID Document
-  * didType - the type of DID to generate.
-      Options: 'nym' or 'uuid' (default: 'nym')
-  * keyType - the type of keys to generate.
-      Options: 'RsaVerificationKey2018' (default: 'RsaVerificationKey2018').
-  * passphrase - the passphrase to use to encrypt the private keys for
-      nym-based DIDs. Set to `null` if the private keys should not be encrypted.
-  * mode - the mode/environment to generate the DID in.
-      Options: 'dev', 'test', 'live' (default: 'dev').
+```js
+// Generate a new DID Document
+veresDriver
+  .generate({didType: 'nym', keyType: 'Ed25519VerificationKey2018'}) // default
+  .then(didDocument => {
+    // A new didDocument is generated. Log it to console
+    console.log('Generated:', JSON.stringify(didDocument, null, 2));
```

Comment:
Since this is markdown can we include what the result of the console.log is here? It is not a necessity but it would be quite nice.

Reply:
I'm not in favor of putting a ~30 line DID document representation in the readme, but I could see linking to a sample somewhere, I've created an issue.  https://github.com/veres-one/did-veres-one/issues/32